Orchestrator/README.md

```diff
@@ -92,9 +92,9 @@ This enables basic intent recognition. For more advanced scenarios follow the st
 
 ## Limitations
 
-**Important**:
-Orchestrator is limited to intents only. Entity definitions are ignored and no entity extraction is performed during recognition.
-Only the *default* base model is available to Orchestrator solutions.
+* Orchestrator is limited to intents only. Entity definitions are ignored and no entity extraction is performed during recognition.
```

Comment:
Should we say something like "Orchestrator is limited to intents only in preview release". ?  Also that entity recognition and multi language support are coming in future releases?

Reply:
As long as we clearly outline the limitations of the "current, preview" release here, we can state in the roadmap the features we plan to add in. We [probably] should not set any feature schedule expectations in this article.